mesheryctl/internal/cli/root/system/start.go

```diff
@@ -96,159 +96,159 @@ func start() error {
 	if err != nil {
 		return errors.Wrap(err, "error processing config")
 	}
-	if(mctlCfg.Contexts["local"].Platform== "docker") 
-	{
+	ctxPlatform := mctlCfg.GetContextContent().Platform
+	if ctxPlatform == "docker" || ctxPlatform == "" {
```

Comment:
We might not want to default to docker. @leecalcote shed some light if possible. I dont think we should do anything or maybe give an error if platform is not set.